packages/material-ui-lab/src/VisuallyHidden/VisuallyHidden.test.js

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { getClasses } from '@material-ui/core/test-utils';
+import createMount from 'test/utils/createMount';
+import describeConformance from '@material-ui/core/test-utils/describeConformance';
+import VisuallyHidden from './VisuallyHidden';
+
+describe('<VisuallyHidden />', () => {
+  const mount = createMount();
+  let classes;
+
+  before(() => {
+    classes = getClasses(<VisuallyHidden />);
+  });
+
+  describeConformance(<VisuallyHidden />, () => ({
+    classes,
+    inheritComponent: 'span',
+    mount,
+    refInstanceof: window.HTMLSpanElement,
+    skip: ['componentProp'],
+  }));
```

Comment:
Should we add more tests for testing specifically that these styles are applied:
```js
    border: 0,
    clip: 'rect(0 0 0 0)',
    height: '1px',
    margin: '-1px',
    overflow: 'hidden',
    padding: 0,
    position: 'absolute',
    whiteSpace: 'nowrap',
    width: '1px',
```